### ISSUE TITLE
@timestamp is converted to local TZ offset (1.4.2)

### ISSUE THREAD

**Issue by talvey** [elastic/logstash#2358]
When logstash reads a message/event with an existing @timestamp field, the @timestamp value is converted to the local timezone offset.  For example, a @timestamp with a value of "2015-01-14T15:51:55.310Z" prior to the logstash input will get converted to "2015-01-14 08:51:55.310-0700" (in my case) after the output.  This format differs from the output of the "date" filter.  My understanding from the documentation is the @timestamp should always be in UTC. This may be fixed in 1.5, but I still thought it was worth noting.

I believe the issue is in the time_addon.rb where the parse_iso8601 creates a new Time (with the local tz) with the following code

```ruby```
Time.at(millis / 1000, (millis % 1000) * 1000)
```
Could that be changed to the following?
```ruby```
Time.at(millis / 1000, (millis % 1000) * 1000).getutc
```

**Comment by jordansissel** [comment]
`.utc` I think, but yes you're right. @timestamp should always be in utc

**Comment by colinsurprenant** [comment]
`@timestamp` handling has been completely reworked in 1.5 and is alway in UTC so this is fixed in 1.5. We could keep this open for if we plan a 1.4.3 release.

**Comment by jordansissel** [comment]
I think we can close this; we don't have plans for 1.4.3 at this time
